src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -641,7 +641,7 @@
 short-summary: Update a firewall rule.
 examples:
   - name: Update a firewall rule
-    text: az sql server firewall-rule update -g mygroup -s myserver -n myrule --start-ip-address 9.8.7.6 --end-ip-address 5.4.3.2
+    text: az sql server firewall-rule update -g mygroup -s myserver -n myrule --start-ip-address 5.4.3.2 --end-ip-address 9.8.7.6
```

Comment:
Just one small question. I'm not familiar with sql. What's the reason for this PR?

Reply:
oh. I got the point.